.github/workflows/main.yml

```diff
@@ -8,8 +8,6 @@ on:
     branches:
       - main
   push:
-    branches:
-      - main
```

Comment:
Why is this removed? Won't this prevent this workflow from running on the main branch when PRs are merged?

Reply:
@ghislainbourgeois, this change will make the workflow run for all branches (not just the `main` branch). I added those two lines last night to follow the same template we are using for all of the other repos. I am ok either way (keeping these two lines or removing them as done in this PR). What do you think? or what is your preference?

Reply:
Oh right, I forgot the `push:` line above. That works for me then!